src/code-templates/libraries/redis/index.ts

```diff
@@ -0,0 +1,87 @@
+import { RedisClientType } from '@redis/client'
+import { ConnectionState, create, RedisOptions } from './redis-client'
+import { RedisResult } from './redis-result'
+
+let client: RedisClientType
+
+let state: ConnectionState = ConnectionState.DISCONNECTED
+
+export const init = async (options: RedisOptions) => {
+  if (client == null) {
+    client = create(options) as RedisClientType
+
+    await client.connect()
+    state = ConnectionState.CONNECTED
+    client.on('error', (err) => {
+      console.log('err', err)
```

Comment:
Look like you forgot to remove the debugging code 😉 

In case we want to log which is perfectly fine I think we have a logger library and maybe add a meaningful message


same for ready log on line 20 😄